client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -209,9 +209,12 @@ def get_export_config(self, instance):
         }
 
         # Create the list of Texture Sets to export.
-        config["exportList"] = []
-        for texture_set in substance_painter.textureset.all_texture_sets():
-            config["exportList"].append({"rootPath": texture_set.name()})
+        export_texture_set = creator_attrs.get("exportTextureSet", [])
```

Comment:
I think we should only set this data if any export texture set filtering is provided. If not, then just not pass it along instead of explicitly passing along all?

Which would also make this code way more readable because when specified you can directly pass the input values.